mp/src/game/client/momentum/ui/MapSelection/MapSelectorDialog.cpp

```diff
@@ -56,7 +56,7 @@ CMapSelectorDialog::CMapSelectorDialog(VPANEL parent) : Frame(nullptr, "CMapSele
     SetParent(parent);
     SetScheme(scheme()->LoadSchemeFromFile("resource/MapSelectorScheme.res", "MapSelectorScheme"));
     SetProportional(true);
-    SetSize(GetScaledVal(680), GetScaledVal(400));
+    SetPos(vgui::scheme()->GetProportionalScaledValue(235), vgui::scheme()->GetProportionalScaledValue(115));
```

Comment:
Any reason to not use `GetScaledVal` here?

Also, why set the pos here if you moved the size to the res file. Shouldn't the pos also be there as well?

Reply:
I didnt see that GetScaleVal was the same thing but neat my bad.
I put setpos in codebecause it works with different aspect ratios so wouldnt need to be changed but there is no reason for it to not be in the .res you are right. 

fixed it up